### DISCUSSION TITLE
Forbid the use of environments and releases within a single helmfile.yaml part

### DISCUSSION THREAD

**Discussion by yxxhero** [helmfile/helmfile#588] [Ideas]
see: https://github.com/helmfile/helmfile/blob/main/docs/proposals/towards-1.0.md

/assign @yxxhero

**Comment by AssafKatz3** [comment]
@yxxhero 
Sorry to be late to the party, but we have a customer who uses double-rendering (even though I completely agree that it is confusing). This is the use case that was discussed at [issue #1875 at the old repository](https://github.com/roboll/helmfile/issues/1875). I couldn't understand if it keep working after this change, but it should be considered a test case for this change.

**Comment by yxxhero** [comment]
@AssafKatz3 PR is welcome. could you add a test case for your case.
